verifier/diff_match_patch.js

```diff
@@ -0,0 +1,55 @@
+var diff_match_patch=function(){this.Diff_Timeout=1;this.Diff_EditCost=4;this.Match_Threshold=.5;this.Match_Distance=1E3;this.Patch_DeleteThreshold=.5;this.Patch_Margin=4;this.Match_MaxBits=32},DIFF_DELETE=-1,DIFF_INSERT=1,DIFF_EQUAL=0;diff_match_patch.Diff=function(a,b){this[0]=a;this[1]=b};diff_match_patch.Diff.prototype.length=2;diff_match_patch.Diff.prototype.toString=function(){return this[0]+","+this[1]};
```

Comment:
Issue: I've had a pull request open against `diff-match-patch` for 7 years to fix its handling of non-BMP Unicode strings, which is currently completely broken. It hasn't been reviewed. There are also 25 more pull requests opened since then. The project does not seem well maintained.

https://github.com/google/diff-match-patch/pull/13